assemble-and-restart

```diff
@@ -71,10 +71,13 @@ if [ ! -z "${ODO_S2I_WORKING_DIR}" ] && [ -n  "$ODO_SRC_BACKUP_DIR" ] && ([ "${O
     rsync -rlO ${ODO_SRC_BACKUP_DIR}/src/. ${ODO_S2I_SRC_BIN_PATH}/src/
 fi
 
-# Now that the assemble is successful; change autostart from false to true in supervisor.conf present on persistent volume mounted on "/opt/app-root"
-PV_MNT_PT="/opt/app-root"
-sed -i 's/autostart=false/autostart=true/g' ${PV_MNT_PT}/conf/supervisor.conf
-
-# Restart supervisord in order to actualy run the application
-# This is a dumb way to restart as supervisord does not have a restart function
-${ODO_UTILS_DIR}/bin/supervisord ctl stop run; ${ODO_UTILS_DIR}/bin/supervisord ctl start run
+# following script need not to run for s2i components converted to devfile
+# for converted components devfile push code assures the following steps
```

Comment:
A little confused by the comment. First line says that the below script should not run for components converted to devfile. Second line says that devfile push code assures below steps for converted components. Sounds opposite to me. Do you think we can reword it?

Reply:
`for components converted to devfile, we don't need to run following steps. As this steps already run by devifile push code`
is this sound ok to you?
``

Reply:
Yep it does. Earlier one made me think like - "these steps don't get executed, but these steps get executed". Maybe my bad. But above sounds perfect. :+1: